QueryBuilder.Test.Generated/QueryBuilder.Test.Generated.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;net6</TargetFrameworks>
```

Comment:
As these are just for holding generated test models to run tests and the test project itself is just net6, I think we can avoid the multiple target framework setup and just stick to net6 here.